esmvalcore/preprocessor/_area.py

```diff
@@ -464,18 +489,21 @@ def extract_shape(cube,
     See Also
     --------
     extract_region : Extract a region from a cube.
-
     """
-
     with fiona.open(shapefile) as geometries:
 
         if crop:
             cube = _crop_cube(cube, *geometries.bounds)
 
-        lon = cube.coord(axis='X').points
-        lat = cube.coord(axis='Y').points
-        if cube.coord(axis='X').ndim == 1 and cube.coord(axis='Y').ndim == 1:
-            lon, lat = np.meshgrid(lon.flat, lat.flat, copy=False)
+        cmor_coords = True
+        natural_earth_file = False
+        if geometries.bounds[0] < 0:
+            cmor_coords = False
+        if "ne_" in str(shapefile):
```

Comment:
I think this is not the best way to test for the fact whether the file does not work with the normal wrap around function even if it only goes for natural earth files (a shape file called coastline_10m.shp would also be picked up). Better would be to check for missing edge coordinates if that is the problem.

Reply:
Also could you add a test for the case that you want to solve here?

Reply:
good point! but unfortunately I am not sure how else to check if it's a Natural Earth file - do you have any clues, maybe an attribute in the metadata or smth? I am not that familiar with NE files :beer: